timescale-cloud/maintenance.md

```diff
@@ -44,6 +44,7 @@ necessary, and sometimes require a short outage.
 Upgrades are performed as rolling upgrades where completely new server instances
 are built alongside the old ones. When the new instances are up and running they
 are synchornized with the old servers, and a controlled automatic failover is
+are synchronized with the old servers, and a controlled automatic failover is
```

Comment:
This looks like a dupe or copy / pasta 😬

Reply:
nope, typo fix 😉